packages/benchmarks/src/implementations/styleq/Dot.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { styleq } from 'styleq';
+
+const Dot = ({ size, x, y, children, color }) => {
+  const [className, inlineStyle] = styleq([
```

Comment:
This basically is the function that would be useful. Tamagui at build time would use this to pre-generate the style object or className, so would want access to it.

Reply:
It's on npm. But `StyleSheet` wraps this function, so you could call `StyleSheet(styles)` to get basically the same result plus logic for RTL polyfill. The output of the RNW compiler could be adjusted to match what a build-time compiler would be able to produce for RTL polyfilling, but I don't know if you're supporting that.

Reply:
The RN API is not great for extensions, but it could be useful to have something like `StyleSheet/runtime` that only exports the code needed to merge the "css" objects (assuming a compiler doesn't leave `StyleSheet.flatten()` calls in the bundle)